.github/workflows/image.yml

```diff
@@ -0,0 +1,86 @@
+name: build
```

Comment:
I would call this action `docker` and the rename the file `docker.yml`

Reply:
addressed